### ISSUE TITLE
Montessori Pull-up (To Stand) Bar 

### ISSUE THREAD

**Issue by nelsonic** [dwyl/home#184]
Content to follow when I get back to my desk …

**Comment by nelsonic** [comment]
![image](https://user-images.githubusercontent.com/194400/134333944-00079665-21b1-4c75-adcc-451a767f3d52.png)

https://www.criancasindependentesmontessori.com/product-page/espelho-com-barra-montessori
![image](https://user-images.githubusercontent.com/194400/134333887-70fc6595-acc1-44a6-b923-788b90252666.png)

**Comment by iteles** [comment]
You had asked about beeswax for coating the bar. 

We have this:
![image](https://user-images.githubusercontent.com/4185328/134365416-b9f4be37-d7f8-42db-8cc5-bf29bd1d9ba4.jpeg)

**Comment by nelsonic** [comment]
As noted in the OP. I opened this issue to capture my mini-project to _build_ a Pull-up-to-Stand Bar. 

Finished reading "**The Montessori Baby**" in August:
https://www.amazon.com/Montessori-Baby-Parents-Nurturing-Understanding/dp/1523512407
![image](https://user-images.githubusercontent.com/194400/136459971-2ed6c102-8a00-411c-b534-98ab10d01a15.png)

_Most_ of the reviews on Amazon are 5-stars. The handful of less favourable reviews miss the point of the book. 
There are other books that cover the "Science" of Montessori: https://www.amazon.com/Montessori-Science-Angeline-Stoll-Lillard/dp/0199981523 this book is a _gentle_ intro for new parents and it's a good "handbook" with practical advice.

> _Highly_ recommend the book to all new/prospective parents (_and grandparents!); It's a **_great_ intro** to **Montessori**. 
> Wish I had read it last year ***`before`*** our son was born, but it was only ***published*** in **March 2021**.
> I had several other baby/parent related books to finish before I could get onto this one. 


![image](https://user-images.githubusercontent.com/194400/136459615-e6e3f631-12af-46d3-94d3-cdf9241334b7.png)

> Yes, this is his hand in the photo: I only opened the book to take this photo
> then put the book away and gave him all the attention he needed, don't worry.

Bar on the wall (_with mirror_) is mentioned several times in the book as a good way for baby to practice lifting/standing and walking along the bar while staring at themself in the mirror:
![image](https://user-images.githubusercontent.com/194400/136459500-3e434e10-7321-4c87-bc0b-67ed7d6bc9b1.png)


He was already able to stand but not lift himself up. This photo was taken when he was placed standing in his floor-bed:
![image](https://user-images.githubusercontent.com/194400/136459170-87ba75f8-3ea8-4db5-8274-2991f4704936.png)

**Comment by nelsonic** [comment]
I figured I could ***`make`*** a decent bar that would fit the space available.
So I took a quick trip to the hardware store and bought:

35mm Pine Dowels x 2400mm (basically what you would use for a curtain rail, but his is un-treated kiln dried pine):
![image](https://user-images.githubusercontent.com/194400/136465934-cc02f8be-f9da-47ee-8849-7793904c1155.png)

Planed pine board: any thickness is fine, as you'll see below. but I went with 18mm thick and 69mm wide, it's what they had:
![image](https://user-images.githubusercontent.com/194400/136466056-2fb863fb-d4f0-444b-93b2-ba2848ee45b1.png)

These are my materials in the cart:
![image](https://user-images.githubusercontent.com/194400/136466229-01fc6eff-8d23-4746-8690-ccf6ddbc56c8.png)

Got a couple of sheets of 10mm construction grade (cheap) plywood for an up-comming project and some other hardware.
But the total material cost for _this_ project was €7.99 x 2 + €6.99 = **`€22.97`**

The 2400mm rods/boards did not fit in our _tiny_ Toyota IQ, so I had to drive with the "boot" open:
<img width="815" alt="Screen Shot 2021-10-07 at 22 43 04" src="https://user-images.githubusercontent.com/194400/136466619-eb3f8ac5-7715-4d47-a973-ddc4b4cf0a7d.png">

It's less than 2km from the hardware store, I did it at 20:30 at night and my lights were still perfectly visible. Low risk.
Made it home safely without any drama:
![image](https://user-images.githubusercontent.com/194400/136466760-345978af-b564-4148-9ff6-ad7ec2c03854.png)

**Comment by nelsonic** [comment]
Cut the boards to size and a few to 45<sup>o</sup> on the Mitre saw:
![image](https://user-images.githubusercontent.com/194400/136466848-dd5a8779-a54f-49e5-821b-2898daa52cc8.png)

These are the 8 pieces for fixing the bars to the wall:
![image](https://user-images.githubusercontent.com/194400/136466891-cf45efd5-733a-4dd0-8411-bf061275429c.png)

Measured where to drill holes:
![image](https://user-images.githubusercontent.com/194400/136466941-8a3b4f08-0a7e-4d94-87b7-594642a0c627.png)

Put these Parkside forstner bits I got at Lidl a while ago to use. (The cost €2.99 which is a steal!)
![image](https://user-images.githubusercontent.com/194400/136467076-1712c695-793c-46fb-b160-b29f2b126b78.png)



Using a forstner bit, I drilled holes in the pieces to receive the dowels:
![image](https://user-images.githubusercontent.com/194400/136467043-92754865-0b89-4929-9044-d4165f0eb560.png)

A hand drill would have worked fine. I invested in a column drill a while back, so I used that. `#Lasers` 😜 

These are the 4 pieces drilled: (but not sanded)
![image](https://user-images.githubusercontent.com/194400/136467299-303bcbd2-64dd-4e68-9c20-8c51fb417475.png)

Measured the diameter of the screw head I'd be using to fix the bars to the walls:
![image](https://user-images.githubusercontent.com/194400/136467373-d813c877-cb0e-4069-9c1e-b5b279cf672f.png)

I drilled out the holes for mounting to the wall and "counter-sunk" them with a 15mm forstner bit:
![image](https://user-images.githubusercontent.com/194400/136467593-0085cecc-bcc6-4fa9-858e-c46985b00927.png)

Then using dowel drilling bits, again bought at Lidl for €2.40:
![image](https://user-images.githubusercontent.com/194400/136467472-29e96ab6-1b78-4696-b533-898a65f6bb7e.png)

I drilled 2 dowels of the same pine material:
![image](https://user-images.githubusercontent.com/194400/136467513-c614a772-b67c-4e34-a423-11c1d41a3de7.png)

Sliced the 2 dowels into 4 using the dremel:
![image](https://user-images.githubusercontent.com/194400/136467839-b0482703-5a93-48fc-817c-3ee3950033a4.png)

Erased all my construction (pencil) lines:
![image](https://user-images.githubusercontent.com/194400/136467887-a985ff87-99ef-43f1-b703-a60aa6f577e0.png)

These are all my mounting pieces before the glue-up:
![image](https://user-images.githubusercontent.com/194400/136467940-9b80c22e-f06d-4f89-9b68-ba648f9d2538.png)

One of the brackets being glued with a G-Clamp:
![image](https://user-images.githubusercontent.com/194400/136467992-89c4589d-1667-4d1a-ae23-f32db0ba8b24.png)

All 4 glueing overnight:
![image](https://user-images.githubusercontent.com/194400/136468047-bd1d6e90-fbaf-40d3-a36a-3f77daaabdf1.png)

At this point you can already see what I'm making. ⏳ 

Then cut the pine rods/dowels down the size I needed and glued them into the mounting brackets:
![image](https://user-images.githubusercontent.com/194400/136468127-b33a72da-59ce-4a6a-8c54-9dd730f86194.png)

While I waited for the glue to dry, I went to measure the holes and confirmed that there weren't any electrical wires behind them:
![image](https://user-images.githubusercontent.com/194400/136468298-a14b74f9-891c-4920-8db1-d8f914d84ea0.png)

Drilled the holes and mounted the two bars:
![image](https://user-images.githubusercontent.com/194400/136468340-715bb424-1e89-465a-8222-163c9b4c6f49.png)

**Comment by nelsonic** [comment]
This is our little angel trying the bars for the first time:
![image](https://user-images.githubusercontent.com/194400/136468461-2821a57a-141a-4385-87cf-46b916108675.png)

He's been enjoying them for the past couple of weeks and he's mastered pulling up to stand and shuffling along them.
We've taken _hundreds_  (no exaggeration) of photos and videos of him using them. 

![image](https://user-images.githubusercontent.com/194400/136468732-f826368a-09b6-4cde-acde-28fe8eb3dc60.png)

This is the final shot with the dowels in to cover the screws and the mirrors in place:
![image](https://user-images.githubusercontent.com/194400/136468876-45a34cb8-5246-4a86-8f60-fada508df5bf.png)

Totally worth €25 in materials + 1h of my time to buy the materials, cut, glue and mount them in place:
![image](https://user-images.githubusercontent.com/194400/136468958-79a1a845-5b50-42f3-8ee7-907cfa74c452.png)

Perfect fit. _Hours_ of fun.

**Comment by nelsonic** [comment]
Yeah, it definitely took longer to *`make`* it with taking all the photos, ⏳ 
and once I was done, I regretted not capturing video. 🎥 
But the setup time for each shot to do video is considerably longer ... 🤔 
and I don't have much _space_ on my iPhone as it is ... 🙄 

And, yes I procrastinated on uploading the photos because I knew it would take a while,
and the photos don't really add anything to the *`function`* of the device, they're just "documentary" ... 
but glad I took them and uploaded. ❤️
